docs/web/using_with_vue/api.mdx

```diff
@@ -0,0 +1,117 @@
+---
+id: api
+title: API
+sidebar_label: API
+slug: /web/using_with_vue/api
+---
+
+## TolgeeProvider
+Provides tolgee instance. Use in root of your application.
+
+```js
+import { TolgeeProvider } from '@tolgee/vue';
+```
+
+```html
+<TolgeeProvider config="...">
+  ... content ...
+</TolgeeProvider>
+```
+
+#### Prop `config`
+[`TolgeeConfiguration`](../configuration) object, which is passed to tolgee instance.
+
+#### Prop `loadingFallback?` 
+`String | JSX.Element` - it is rendered when tolgee is loading translations instead of provided content.
+You can pass custom loading element when using Vue with JSX.
+
+#### Slot `fallback`
+Alternative for `loadingFallback` use when you are using Vue with templates:
```

Comment:
usage?

Reply:
I think it's explainded above - in `loadingFallback`